src/createSigasiProject.py

```diff
@@ -3,11 +3,13 @@
     :license: BSD, see LICENSE for more details.
 """
 from SigasiProjectCreator import ConverterHelper
-from SigasiProjectCreator.DotF import DotFfileParser
+from SigasiProjectCreator.ArgsAndFileParser import ArgsAndFileParser
 
 
 def main():
-    ConverterHelper.parse_and_create_project(DotFfileParser.usage, DotFfileParser.parse_file)
+    parser = ArgsAndFileParser()
+    parser.parse_args()
+    ConverterHelper.parse_and_create_project()
```

Comment:
This should be:

```python
parser = ArgsAndFileParser()
parser.parse_args()
ConverterHelper.parse_and_create_project(parser.options)
```

The parser should not have a class variable storing its results.

If too many of the functions in the ConverterHelper use the options to pass them around, the options could be passed to a constructor.

```python
parser = ArgsAndFileParser()
parser.parse_args()
ConverterHelper(parser.options).parse_and_create_project()
```

(note: python modules should have [lowercase](https://peps.python.org/pep-0008/#package-and-module-names) names)

Reply:
Rewritten as

```
def main():
    ProjectCreator(ProjectOptions()).create_project()
```

Lowercasing the module name is for later, if you don't mind